docs/development/testguide.rst

```diff
@@ -680,15 +680,51 @@ block::
 Image tests with pytest-mpl
 ===========================
 
+Running image tests
+-------------------
+
 We make use of the `pytest-mpl <https://pypi.python.org/pypi/pytest-mpl>`_
-plugin to create tests where we can compare the output of plotting commands
-with reference files (this is used for instance in
+plugin to write tests where we can compare the output of plotting commands
+with reference files on a pixel-by-pixel basis (this is used for instance in
 :ref:`astropy.visualization.wcsaxes <wcsaxes>`).
 
 To run the Astropy tests with the image comparison, use::
 
     python setup.py test -a "--mpl" --remote-data
 
+However, note that the output can be very sensitive to the version of Matplotlib
+as well as all its dependencies (e.g. freetype), so we recommend running the
+image tests inside a `Docker <https://www.docker.com/>`__ container which has a
+frozen set of package versions (Docker containers can be thought of as mini
+virtual machines). We have made a set of Docker container images that can be
+used for this. Once you have installed Docker, to run the Astropy tests with the
+image comparison inside a Docker container, make sure you are inside the Astropy
+repository (or the repository of the package you are testing) then do::
+
+    docker run -it -v ${PWD}:/repo astrofrog/image-tests-py35-mpl153:1.0 /bin/bash
```

Comment:
I guess, this needs to be updated from astrofrog to astropy

Reply:
Done